.gitignore

```diff
@@ -0,0 +1,2 @@
+**/*.pyc
+.idea/**
```

Comment:
the pyc ignore is consistent with the project, but idea is your IDE, so this kind of ignore (ignoring .idea folder) must not exist here, but in your personal gitignore